app/assets/src/components/views/amr_heatmap/AMRHeatmapControls.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Divider } from "~/components/layout";
+import { Dropdown } from "~ui/controls/dropdowns";
+
+import cs from "./amr_heatmap_view.scss";
+
+export default class AMRHeatmapControls extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  generateChangeFunction(filter) {
+    const changeFunction = option => {
+      if (option === this.props.selectedOptions[filter]) {
+        return;
+      }
+      this.props.onSelectedOptionsChange({ [filter]: option });
+    };
+    return changeFunction;
+  }
+
+  renderFilterDropdowns() {
+    const filtersList = [];
+    this.props.filters.forEach((_, filter) => {
+      filtersList.push(
+        <div className="col s3">
+          <Dropdown
+            fluid
+            rounded
+            options={this.props.filters.get(filter).options}
```

Comment:
since you're already iterating over `filter`, you can just do `filter.options`, right?

Reply:
`filter` is a key here, not the whole entry, but yeah I could just use the entry (which is the `_` parameter I didn't use). Updated